scripts/ci/jobs/diff-dumps.sh

```diff
@@ -122,12 +122,12 @@ upload_offline_dump() {
 
     file_to_check="scanner-v4-defs-${latest_version}.zip"
 
-    if curl --silent --show-error --max-time 60 --retry 3 -o $file_to_check https://definitions.stackrox.io/v4/offline-bundles/$file_to_check; then
+    if curl --silent --show-error --max-time 60 --retry 3 -o $file_to_check https://definitions.stackrox.io/v4/offline-bundles/$file_to_check && jq . $file_to_check; then
```

Comment:
Use `--fail` and you don't need the `if-else`.

Reply:
There is also `--fail-with-body`.

Reply:
but  `--fail` doesn't help to exit the process with status 1

Reply:
Never mind, I saw this command: `set -euo pipefail`